README.md

```diff
@@ -1,2 +1,107 @@
+<img src="http://www.braveterry.com/wp-content/uploads/2015/03/swagger2.png"/>
+
 # cowboy-swagger
-Swagger integration for Cowboy (built on trails)
+[Swagger](http://swagger.io/) integration for [Cowboy](https://github.com/ninenines/cowboy) (built on [trails](https://github.com/inaka/cowboy-trails)).
+
+## Contact Us
+For **questions** or **general comments** regarding the use of this library,
+please use our public [hipchat room](https://www.hipchat.com/gpBpW3SsT).
+
+If you find any **bugs** or have a **problem** while using this library, please
+[open an issue](https://github.com/inaka/elvis/issues/new) in this repo
+(or a pull request :)).
+
+And you can check all of our open-source projects at [inaka.github.io](http://inaka.github.io).
+
+## Why Cowboy Swagger?
+Simple, because there isn't a tool in Erlang to document Cowboy RESTful APIs, easy and fast,
+improving development productivity.
+
+With `cowboy_swagger` is possible to integrate Swagger to your Erlang projects, that use Cowboy as web server.
```

Comment:
...to integrate Swagger to your Erlang projects that use Cowboy as a web server.